marklogic-data-hub-central/build.gradle

```diff
@@ -79,7 +79,7 @@ dependencies {
     implementation "com.squareup.okhttp3:logging-interceptor:4.7.2"
 
     // Excluding okhttp3 dependency as it bumps down the dependency from 4.4.0 to 3.x
-    implementation ('com.microsoft.azure:azure-keyvault-secrets-spring-boot-starter:2.2.4') {
+    implementation ('com.microsoft.azure:azure-keyvault-secrets-spring-boot-starter:2.3.5') {
         exclude group: "com.squareup.okhttp3", module: "okhttp"
     }
```

Comment:
I'm working on https://project.marklogic.com/jira/browse/DHFPROD-6397 and as part of that I'll test out these changes on Azure as well.